src/theme.js

```diff
@@ -1,4 +1,20 @@
-var jQuery = (typeof(window) != 'undefined') ? window.jQuery : require('jquery');
+/**
+ * Inject jquery if isn't present already.
+ *
+ * @param {function} fn - Function to be executed after jquery has been loaded.
+ */
+function injectJQuery(fn) {
+    if (!window.jQuery) {
+        console.log("JQuery not found. Injecting.");
+        var script = document.createElement("script");
+        script.type = 'text/javascript';
+        script.src = "https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js";
```

Comment:
I probably wouldn't refer to a Google hosted file here, and instead would bundle a jQuery release with the theme.

Reply:
So, we just bundle jquery with the theme instead of conditionally injecting it?

Reply:
@ericholscher was +1 on injecting it conditionally.

Reply:
I think it's good to use  CDN. However, I'd point it to the official one instead of Google, as I did in sphinx-hoverxref: https://github.com/readthedocs/sphinx-hoverxref/pull/204/files#diff-702aaa7a7fb4adbe9dd9ce74f658c943aa3321b24c88d959d5a838761bb6c902R331

Official docs: https://jquery.com/download/#using-jquery-with-a-cdn

Reply:
> So, we just bundle jquery with the theme instead of conditionally injecting it?

Yup. The concerns here are privacy, security, and creating a dependency downstream on a third party service. Package maintainers likely need the file locally as well.

The historical benefits of hosting JS directly from a CDN are no longer all relevant, and probably don't outweigh the privacy/security negatives this brings:

https://httptoolkit.tech/blog/public-cdn-risks/

Reply:
Changed to bundle jquery with the theme